Studies/source/PuritySignalStudy.cxx

```diff
@@ -66,10 +66,14 @@ class SignalPurityStudy {
         auto file = root_ext::OpenRootFile(args.inputPath());
         auto tuple = ntuple::CreateEventTuple("all_events", file.get(), true, ntuple::TreeState::Full);
 
-        int count = 0;
+        taus_legs = { {LegType::e, false},
+                      {LegType::mu, false},
+                      {LegType::tau, true} };
+
+        lep_wp_deep_tau = { {Channel::ETau,  0.0630386},
```

Comment:
use working points stored in the ntuple, cuts on raw value was temporary solution

Reply:
`lep_wp_deep_tau` is no longer used. Remove it.